ultimatepython/data_structures/linkedlist.py

```diff
@@ -0,0 +1,117 @@
+class Node:
+    def __init__(self, dataval):
+        self.dataval = dataval
+        self.nextval = None
+        self.preval = None
+
+class CLinkedList:
```

Comment:
What is the rationale for this class being called `CLinkedList` instead of simply `LinkedList`?